src/framework/application.js

```diff
@@ -457,6 +458,21 @@ function Application(canvas, options) {
         id: LAYERID_WORLD
     });
 
+    // placeholder for area light luts
+    var placeholderLutTex =  new Texture(this.graphicsDevice, {
+        width: 2,
+        height: 2,
+        format: PIXELFORMAT_R8_G8_B8_A8
+    });
+    placeholderLutTex.name = 'placeholder';
+
+    var pixels = placeholderLutTex.lock();
+    pixels.fill(0);
+    placeholderLutTex.unlock();
+
+    this.graphicsDevice.scope.resolve('areaLightsLutTex1').setValue(placeholderLutTex);
+    this.graphicsDevice.scope.resolve('areaLightsLutTex2').setValue(placeholderLutTex);
```

Comment:
I'd vote for moving this to a function as well

Reply:
This doesn't seem right to me. The `Application` is just an abstraction of a PlayCanvas app. It shouldn't know bout setting of uniforms etc. Seems like the data should be prepared+set in the `ForwardRenderer`. We might have another renderer one day that can't use the LUT data in the same way.

Reply:
hmm - yes - I will change it so that project settings and asset handling is in Application and placeholder and real LUT textures are initialized and owned by the renderer.

Reply:
I moved texture and uniform setting from Application into ForwardRenderer - I noticed there are some other places in application.js that perhaps should be refactored in this way too - so will add an issue to deal with this in another PR.